src/style.css

```diff
@@ -93,6 +104,294 @@ header {
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: var(--color-white);
+}
+
+.button-list {
+  display: flex;
+  align-items: center;
+  padding: 0;
+  margin: 12px 75px 22px 16px;
+  gap: 12px;
+  width: 236px;
+  height: 40px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: var(--color-white);
+}
+
+.button-list button {
+  display: flex;
+  align-items: center;
+  padding: 10px 12px;
+  height: 40px;
+  background-color: rgba(255, 255, 255, 0.24);
+  border: 1px solid rgba(255, 255, 255, 0.24);
+}
+
+.about-me button {
+  margin-top: 24px;
+  margin-right: 175px;
+  padding: 12px;
+  width: 161px;
+  height: 48px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  background-color: var(--color-Safety-Orange);
+  color: var(--color-white);
+}
+
+.button button:hover {
+  color: #ff6b00;
+  background: #fff5e1;
+  border: 1px solid var(--color-Safety-Orange);
+}
+
+.button button:active {
+  color: #f4f1f0;
+}
+
+.Recent-work:nth-child(1) .button button {
+  padding: 10px 12px;
+  background-color: var(--color-white);
+  border: 1px solid #8993a4;
+}
+
+.Recent-work .see-project-btn,
+.card .see-project-btn {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 700;
+  font-size: 17px;
+  line-height: 24px;
+  letter-spacing: 0.03em;
+  color: var(--color-white);
+}
+
+.Recent-work:nth-child(1) .button {
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 15px;
+  line-height: 20px;
+  color: #091e42;
+}
+
+.Recent-work .see-project-btn {
+  display: flex;
+  align-items: center;
+  margin-top: 12px;
+  margin-bottom: 24px;
+  border: 1px solid var(--color-Safety-Orange);
+  padding: 12px;
+  width: 124px;
+  height: 48px;
+  background-color: var(--color-Safety-Orange);
+}
+
+/* work card style  */
+
+.work-cards {
+  display: grid;
+  grid-template-columns: 327px;
+  grid-template-rows: repeat(6, 386px);
+}
+
+.card {
+  display: flex;
+  flex-flow: wrap column;
+  justify-content: end;
+  background-image: url("./images/background-3.svg");
+  background-repeat: no-repeat;
+}
+
+.card h3 {
+  height: 81px;
+  font-family: "Crete Round", serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 32px;
+  line-height: 44px;
+  color: var(--color-white);
+  margin: 0 16px 12px 16px;
+}
+
+.description {
+  height: 87px;
+  font-family: "Inter", sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  color: var(--color-white);
+  margin: 0 16px;
+}
+
+.card .see-project-btn {
+  width: 327px;
+  height: 48px;
+  margin-top: 22px;
+  background-color: var(--color-Safety-Orange);
+  border: 1px solid var(--color-Safety-Orange);
+}
```

Comment:
Kindly ensure that the project technologies are designed similarly to that of the figma design. The font color and background colour do not match completely.

Your design:
![image](https://user-images.githubusercontent.com/20554583/213451749-4d510456-2b85-4dfb-b28f-fc950fe10358.png)

Figma design:
![image](https://user-images.githubusercontent.com/20554583/213451807-25fd1b56-5087-4c27-851f-64b92ca35101.png)

Reply:
resolve this